opslevel/resource_opslevel_integration_aws.go

```diff
@@ -79,18 +80,22 @@ func (r *IntegrationAwsResource) Schema(ctx context.Context, req resource.Schema
 			},
 			"ownership_tag_keys": schema.ListAttribute{
 				ElementType: types.StringType,
-				Description: "Allow tags imported from AWS to override ownership set in OpsLevel directly",
+				Description: "Allow tags imported from AWS to override ownership set in OpsLevel directly. Max 5 (default = [\"owner\"])",
 				Optional:    true,
 				Computed:    true,
-				Default:     listdefault.StaticValue(types.ListValueMust(types.StringType, []attr.Value{types.StringValue("owner")})),
+				// current API default below
+				Default: listdefault.StaticValue(types.ListValueMust(types.StringType, []attr.Value{types.StringValue("owner")})),
```

Comment:
Lets make sure we test this when AWS was created with 1.2.x or 1.0.x  and see if it upgrades properly or if we need a state updater?